CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 4.5.0 (2021-01-19)
+
+- Added modern theme styles for the vertical tabset component. [#150](https://github.com/blackbaud/skyux-tabs/pull/150)
```

Comment:
```suggestion
- Added modern theme styles for the vertical tabs component. [#150](https://github.com/blackbaud/skyux-tabs/pull/150)
```
The docs refer to "tabs" instead of "tabset," so it seems like we should do the same here unless the docs are wrong.